### ISSUE TITLE
kerning-pairs not available in current release build

### ISSUE THREAD

**Issue by NworbLegin** [vladimirgamalyan/fontbm#23]
Is there a chance that you could build a new executable release with the kerning-pairs feature in please?

The current build was before the kerning-pairs feature was added

Cheers

Nigel

**Comment by vladimirgamalyan** [comment]
Hi!
Just released the newest [Release](https://github.com/vladimirgamalyan/fontbm/releases/tag/v0.6.1), hope it will work =)

**Comment by NworbLegin** [comment]
Many thanks

 

Cheers

 

Nigel

**Comment by NworbLegin** [comment]
Yep - seems to be working correctly when I set it to `-kerning-pairs extended`, but I didn't get any kerning values with `basic` or `regular` when testing with Google's NotoSans-Regular.ttf

**Comment by vladimirgamalyan** [comment]
It seems Freetype somehow can't read kerning information from the font file (due limitations or bugs) and there are no kernings at all from fontbm perspective. 
"extended" option adds extra synthesized kernings, you can see with `-kerning-pairs extended`.
